test/system/800-config.bats

```diff
@@ -91,6 +91,7 @@ EOF
     cat > $conf_tmp <<EOF
 [containers]
 annotations=['module=$random_data']
+env_host=true
```

Comment:
Why this change? Does not look like you test this anywhere.

Reply:
Artifact...I first wanted to move the check into this test.  Thanks for catching.